### PR TITLE
Fetch krankerl before running it

### PR DESCRIPTION
**Pull request by blizzz**

### CODE REVIEW

File: workflow-templates/appstore-build-publish.yml [hunk 1]
```diff
@@ -90,6 +90,11 @@ jobs:
           cd ${{ env.APP_NAME }}
           npm ci
           npm run build
+          
+      - name: Fetch krankerl
+        run: |
+          wget https://github.com/ChristophWurst/krankerl/releases/download/v0.13.2/krankerl
+          chmod +x krankerl
```
Comment (nickvergessen) [inline workflow-templates/appstore-build-publish.yml:97]: needs a `../krankerl` in the command afterwards as we cd there

**Comment by blizzz** [comment]
still fails with  `krankerl: error while loading shared libraries: libgit2.so.1.1: cannot open shared object file: No such file or directory`

(seeing the same by either fetching the binary or installing from deb)

cc @ChristophWurst

**Comment by ChristophWurst** [comment]
> still fails with `krankerl: error while loading shared libraries: libgit2.so.1.1: cannot open shared object file: No such file or directory`

It works on `ubuntu-18.04`. I'll have to look into why ubuntu 20.04 doesn't work at the moment. maybe a dependency is missing from the deb

**Review by skjnldsv — APPROVED** [review]

**Comment by skjnldsv** [comment]
> > still fails with `krankerl: error while loading shared libraries: libgit2.so.1.1: cannot open shared object file: No such file or directory`
> 
> It works on `ubuntu-18.04`. I'll have to look into why ubuntu 20.04 doesn't work at the moment. maybe a dependency is missing from the deb

done

**Pull request merged**